src/resources/SearchAnalysis/SearchAnalysis.ts

```diff
@@ -0,0 +1,15 @@
+import dayjs from 'dayjs';
+import API from '../../APICore.js';
+import Resource from '../Resource.js';
+import {ReplayAnalysis} from './SearchAnalysisInterface.js';
+
+export default class SearchAnalysis extends Resource {
+    static baseUrl = '/rest/search/v3/analysis';
+
+    replay(id: string, from = dayjs().subtract(1, 'week').format('YYYY-MM-DD')) {
```

Comment:
Implementation is incomplete.
https://platform.cloud.coveo.com/docs?urls.primaryName=Search%20API#/Analysis%20V3/replay
There's a `to` parameter too.

When we implement a route in the platform-client.

Reply:
- [ ] JSDOC, especially when there's date involved.

Reply:
Hmmm, seems like it's in the JSON, but Swagger isn't displaying any of the descriptions 😞 
Seems to be the case for many APIs in fact...
I'll have to dig deeper. In the meantime, the descriptions show up correctly on the docs site: https://docs.coveo.com/en/13/api-reference/search-api#tag/Analysis-V3/operation/replay